scripts/screens/Screens.py

```diff
@@ -906,6 +906,12 @@ def theme(self) -> str:
                 _settings = ujson.loads(read_file.read())
                 return "dark" if _settings["dark mode"] else "light"
 
+    def update_previous_next_cat_buttons(self) -> None:
+        """Updates disabled status of previous and next cat buttons."""
+        # pylint: disable=no-member
+        self.previous_cat_button.enable() if self.previous_cat else self.previous_cat_button.disable()
+        self.next_cat_button.enable() if self.next_cat else self.next_cat_button.enable()
```

Comment:
```suggestion
        self.previous_cat_button.enable() if getattr(self, "previous_cat") else self.previous_cat_button.disable()
        self.next_cat_button.enable() if getattr(self, "next_cat") else self.next_cat_button.disable()
```
Just so it won't throw a crash if it's called on a screen that doesn't have the `previous_cat` and `next_cat` attributes. Also so that we actually do disable the `next_cat` button if it's invalid.

Reply:
Pretty sure you meant hasattr and not getattr, getattr throws an error when the attribute doesn't exist.
Also, I've changed it to `hasattr(self, "previous_cat") and self.previous_cat` since short-circuiting means that it won't throw an error (I believe)